pages/common/scrot.md

```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Capture screenshots using the command line.
+
+- Capture the whole screen:
+
+`scrot`
+
+- Select a window with a mouse then capture the window:
```

Comment:
`a mouse`? Does that mean there's more than one cursor one the screen? :stuck_out_tongue: :-)